### PR TITLE
add EXPLAIN ANALYZE fields; update follower read example

### PR DESCRIPTION
**Pull request by taroface**
DOC-10507
DOC-10506
DOC-10396
DOC-10395

- Updated EXPLAIN ANALYZE tables and example output with new fields in 24.3
- Added some fields that were not previously documented
- Added example to Follower Reads doc

### CODE REVIEW

File: src/current/v24.3/follower-reads.md [hunk 1]
```diff
@@ -175,9 +173,61 @@ This example performs a bounded staleness follower read against a [demo cluster]
     (7 rows)
     ~~~
 
-### Verify that a cluster is performing follower reads
+### Verify that CockroachDB is performing follower reads
+
+To verify that a cluster is performing follower reads, go to the [**Custom Chart** debug page in the DB Console]({% link {{ page.version.version }}/ui-custom-chart-debug-page.md %}) and add the metric `follower_read.success_count` to the time-series graph. The number of follower reads performed by your cluster will be shown.
```
Comment (yuzefovich) [inline src/current/v24.3/follower-reads.md:178]: The metric name is `follower_reads.success_count` (note `s` after `read`).

**Review by yuzefovich — APPROVED** [review]
Nice, thanks! Just one comment from me.

File: src/current/v24.3/follower-reads.md (hunk 1 above)
Comment (taroface) [reply src/current/v24.3/follower-reads.md:178]: Fixed!

File: src/current/v24.3/explain-analyze.md [hunk 2]
```diff
@@ -67,24 +67,30 @@ Property        | Description
...
 `regions` | The [regions]({% link {{ page.version.version }}/show-regions.md %}) where the affected nodes were located.
+`sql cpu time` | The total amount of time spent in the [SQL layer]({% link {{ page.version.version }}/architecture/sql-layer.md %}). It does not include time spent in the [storage layer]({% link {{ page.version.version }}/architecture/storage-layer.md %}).
 `max sql temp disk usage` | ([`DISTSQL`](#distsql-option) option only) How much disk spilling occurs when executing a query. This property is displayed only when the disk usage is greater than zero.
 `estimated RUs consumed` | The estimated number of [Request Units (RUs)]({% link cockroachcloud/plan-your-cluster-basic.md %}#request-units) consumed by the statement. This property is visible only on CockroachDB {{ site.data.products.basic }} clusters.
+`isolation level` | The [isolation level]({% link {{ page.version.version }}/transactions.md %}#isolation-levels) at which this statement executed.
```
Comment (florence-crl) [inline src/current/v24.3/explain-analyze.md:81]: nit while you are adding new properties, would you be able to add `priority`? Using `cockroach demo`, I saw it listed:

```
demo@127.0.0.1:26257/movr> explain analyze select * from rides AS OF SYSTEM TIME '-5s';
                                      info
---------------------------------------------------------------------------------
  planning time: 885µs
  execution time: 2ms
  distribution: full
  vectorized: true
  plan type: custom
  rows decoded from KV: 500 (86 KiB, 1 gRPC calls)
  cumulative time spent in KV: 2ms
  maximum memory usage: 250 KiB
  network usage: 0 B (0 messages)
  regions: us-east1
  sql cpu time: 610µs
  isolation level: serializable
  priority: normal
  quality of service: regular
  historical: AS OF SYSTEM TIME 2024-11-21 17:59:02.823835
```

File: src/current/v24.3/follower-reads.md [hunk 3]
```diff
@@ -175,9 +173,61 @@ This example performs a bounded staleness follower read against a [demo cluster]
...
+    ~~~ sql
+    \demo ls
+    ~~~
+
+    The output will list the connection parameters for each node:
+
+    ~~~
+    node 3:
+    ...
+       (sql)      postgresql://demo:demo37199@127.0.0.1:26259/movr?options=-ccluster%3Ddemoapp&sslmode=equire&sslrootcert=%2FUsers%2Fuser%2F.cockroach-demo%2Fca.crt
```
Comment (florence-crl) [inline src/current/v24.3/follower-reads.md:194]: fyi, you could also use the `(cli)` info:

```
demo@127.0.0.1:26257/movr> \demo ls
node 1:
   (cli)      cockroach sql --certs-dir=/Users/florencemorris/.cockroach-demo -u demo -d movr
   (sql)      postgresql://demo:demo9242@127.0.0.1:26257/movr?sslmode=require&sslrootcert=%2FUsers%2Fflorencemorris%2F.cockroach-demo%2Fca.crt

node 2:
   (cli)      cockroach sql --certs-dir=/Users/florencemorris/.cockroach-demo -u demo -p 26258 -d movr
   (sql)      postgresql://demo:demo9242@127.0.0.1:26258/movr?sslmode=require&sslrootcert=%2FUsers%2Fflorencemorris%2F.cockroach-demo%2Fca.crt

node 3:
   (cli)      cockroach sql --certs-dir=/Users/florencemorris/.cockroach-demo -u demo -p 26259 -d movr
   (sql)      postgresql://demo:demo9242@127.0.0.1:26259/movr?sslmode=require&sslrootcert=%2FUsers%2Fflorencemorris%2F.cockroach-demo%2Fca.crt
```

File: src/current/v24.3/follower-reads.md [hunk 4]
```diff
@@ -175,9 +173,61 @@ This example performs a bounded staleness follower read against a [demo cluster]
...
+    ~~~
+
+1. Open a new terminal and open the SQL shell on a non-leaseholder node. For example:
```
Comment (florence-crl) [inline src/current/v24.3/follower-reads.md:197]: How do you know which is a non-leaseholder node?

**Review by florence-crl — APPROVED** [review]
lgtm pending 1 question

File: src/current/v24.3/follower-reads.md (hunk 4 above)
Comment (taroface) [reply src/current/v24.3/follower-reads.md:197]: @yuzefovich I just picked one at random, but is there a straightforward way to tell this? ^

File: src/current/v24.3/explain-analyze.md (hunk 2 above)
Comment (taroface) [reply src/current/v24.3/explain-analyze.md:81]: Added!

File: src/current/v24.3/follower-reads.md (hunk 4 above)
Comment (taroface) [reply src/current/v24.3/follower-reads.md:197]: @florence-crl I don't think there is an easy way to tell, tbh ... if there is, I will document it in a separate PR!

Comment (yuzefovich) [reply src/current/v24.3/follower-reads.md:197]: You can query `SELECT lease_holder FROM [SHOW RANGES FROM TABLE promo_codes WITH DETAILS];` to get the leaseholder for all ranges of the table called `promo_codes`.

**Pull request merged**